content/en/docs/concepts/architecture/nodes.md

```diff
@@ -513,6 +513,30 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
   recovered since the user was the one who originally added the taint.
 {{< /note >}}
 
+### Forced storage detach on timeout {#storage-force-detach-on-timeout}
+
+In any situation where a pod deletion has not succeeded for 6 minutes, kubernetes will
+force detach volumes being unmounted if the node is unhealthy at that instant. Any
+workload still running on the node that uses a force-detached volume will cause a
+violation of the
+[CSI specification](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerunpublishvolume),
+which states that ControllerUnpublishVolume "MUST be called after all
+NodeUnstageVolume and NodeUnpublishVolume on the volume are called and succeed".
+In such circumstances, volumes on the node in question might encounter data corruption.
+
+With the advent of the "Non-graceful node shutdown" feature, users might opt to disable
+this behavior.
```

Comment:
Try:
```suggestion
The forced storage detach behaviour is optional.
```

Reply:
I've reworded this a bit.

The point is to suggest that folks use "Non-graceful node shutdown" instead of this.

Reply:
Please feel free to take another look.